aspnetcore/tutorials/signalr.md

```diff
@@ -72,9 +72,9 @@ At the end, you'll have a working chat app:
 
 * From the menu, select **File > New Solution**.
 
-* Select **.NET Core > App > ASP.NET Core Web App** (Don't select **ASP.NET Core Web App (MVC)**).
+* Select **.NET Core > App > Web Application** (Don't select **Web Application (MVC)**).
 
```

Comment:
@tdykstra  : Missing a ", then select **Next**" .  So this should read:  Select **.NET Core > App > Web Application** (Don't select **Web Application (MVC)**), then select **Next**."       (Then you get the new dialog page for selecting the target framework mentioned in the next line.)